src/panels/logbook/ha-logbook.ts

```diff
@@ -0,0 +1,156 @@
+import "@polymer/iron-icon/iron-icon";
+
+import formatTime from "../../common/datetime/format_time";
+import formatDate from "../../common/datetime/format_date";
+import { domainIcon } from "../../common/entity/domain_icon";
+import { computeRTL } from "../../common/util/compute_rtl";
+import {
+  LitElement,
+  html,
+  property,
+  TemplateResult,
+  CSSResult,
+  css,
+  PropertyValues,
+} from "lit-element";
+import { HomeAssistant } from "../../types";
+import { fireEvent } from "../../common/dom/fire_event";
+import "lit-virtualizer";
+import { LogbookEntry } from "../../data/logbook";
+
+class HaLogbook extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public entries: LogbookEntry[] = [];
+  @property({ attribute: "rtl", type: Boolean, reflect: true })
+  // @ts-ignore
+  private _rtl = false;
+
+  protected updated(changedProps: PropertyValues) {
+    super.updated(changedProps);
+    if (!changedProps.has("hass")) {
+      return;
+    }
+    const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
+    if (oldHass && oldHass.language !== this.hass.language) {
+      this._rtl = computeRTL(this.hass);
+    }
+  }
+
+  protected firstUpdated(changedProps: PropertyValues) {
+    super.firstUpdated(changedProps);
+    this._rtl = computeRTL(this.hass);
+  }
+
+  protected render(): TemplateResult | void {
+    if (!this.entries.length) {
```

Comment:
` if (!this.entries?.length) {`
Don't forget 😃 

_This was needed on my system at least_